src/uct/api/v2/uct_v2.h

```diff
@@ -197,7 +197,8 @@ typedef struct {
 typedef enum {
     UCT_MD_MEM_REG_FIELD_FLAGS         = UCS_BIT(0),
     UCT_MD_MEM_REG_FIELD_DMABUF_FD     = UCS_BIT(1),
-    UCT_MD_MEM_REG_FIELD_DMABUF_OFFSET = UCS_BIT(2)
+    UCT_MD_MEM_REG_FIELD_DMABUF_OFFSET = UCS_BIT(2),
+    UCT_MD_MEM_REG_FIELD_DEVICE_MEMORY = UCS_BIT(3)
```

Comment:
Should changes in this file to the [API PR](https://github.com/openucx/ucx/pull/9118)